CHANGELOG.md

```diff
@@ -1,4 +1,30 @@
 ## 3.24.0 (Unreleased)
+
+FEATURES:
+* **New Data Source:** `google_secret_manager_secret_version` ([#6432](https://github.com/terraform-providers/terraform-provider-google/pull/6432))
+* **New Resource:** `google_data_catalog_entry_group_iam_binding` ([#6438](https://github.com/terraform-providers/terraform-provider-google/pull/6438))
+* **New Resource:** `google_data_catalog_entry_group_iam_member` ([#6438](https://github.com/terraform-providers/terraform-provider-google/pull/6438))
+* **New Resource:** `google_data_catalog_entry_group_iam_policy` ([#6438](https://github.com/terraform-providers/terraform-provider-google/pull/6438))
+* **New Resource:** `google_data_catalog_entry_group` ([#6438](https://github.com/terraform-providers/terraform-provider-google/pull/6438))
+* **New Resource:** `google_data_catalog_entry` ([#6444](https://github.com/terraform-providers/terraform-provider-google/pull/6444))
+* **New Resource:** `google_dns_policy` is now GA ([#6439](https://github.com/terraform-providers/terraform-provider-google/pull/6439))
+* **New Resource:** `google_secret_manager_secret` ([#6432](https://github.com/terraform-providers/terraform-provider-google/pull/6432))
+* **New Resource:** `google_secret_manager_secret_iam_*` ([#6432](https://github.com/terraform-providers/terraform-provider-google/pull/6432))
```

Comment:
It's not that important to me but maybe we could collapse google_data_catalog_entry_group_iam_member entries similarly or expand this one?